Library/Homebrew/cmd/install.rb

```diff
@@ -32,6 +32,9 @@ def install_args
 
         Unless `HOMEBREW_NO_INSTALL_UPGRADE` is set, `brew install <formula>` will upgrade <formula> if it
         is already installed but outdated.
+
+        Unless `HOMEBREW_NO_INSTALL_FROM_API` is set, `brew install` will ignore your locally edited formula
+        done using `brew edit`.
```

Comment:
I think this should be documented in `brew edit` instead. Most users of `brew install` probably aren't interested in this information, but all users of `brew edit` will probably need to know it.

Maybe `brew edit` can print a warning that can be silenced with `HOMEBREW_NO_ENV_HINTS`?

Reply:
I have removed the message from `install` and added the warning in `edit` in 34a19cf.  
I still think the FAQ should be updated since the command is flat-out wrong; I suppose people might notice the warning when they run `brew edit` but still...

But let me know if you'd like me to / feel free to drop [14927bb](https://github.com/Homebrew/brew/pull/14570/commits/14927bb5955f60f10ff6d88d6e8f33b6c035f0ee)